src/main/webapp/scripts/script.js

```diff
@@ -0,0 +1,6 @@
+const API_KEY = '';
+
+function updateDom(text, containerId) {
+  const containerDOM = document.getElementById(containerId);
+  containerDOM.innerText = text;
+}
```

Comment:
I added this to script.js because several pages might want this helper function and to access the API key

Reply:
Yap looks helpful!